server/server_main.cc

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2020-2021 the Mondradiko contributors.
 // SPDX-License-Identifier: LGPL-3.0-or-later
 
-#include <chrono>
+#include <chrono> // NOLINT [build/c++11]
```

Comment:
![image](https://user-images.githubusercontent.com/94753/107731371-d604e680-6cc3-11eb-8e72-7b356e40332c.png)

apparently this cpplint warning is specific to chromium: https://stackoverflow.com/questions/33653326/google-style-guide-chrono-is-an-unapproved-c11-header